Remotion/Core/Xml/SchemaLoaderBase.cs

```diff
@@ -44,14 +44,14 @@ protected XmlSchema LoadSchema (string schemaFileName)
       Type type = GetType ();
       Assembly assembly = type.Assembly;
 
-      using (Stream schemaStream = assembly.GetManifestResourceStream (type, schemaFileName))
+      using (Stream? schemaStream = assembly.GetManifestResourceStream (type, schemaFileName))
       {
         if (schemaStream == null)
           throw new ApplicationException (string.Format ("Error loading schema resource '{0}' from assembly '{1}'.", schemaFileName, assembly.FullName));
 
         using (XmlReader xmlReader = XmlReader.Create (schemaStream))
         {
-          return XmlSchema.Read (xmlReader, null);
+          return XmlSchema.Read (xmlReader, null)!;
```

Comment:
msdn says not-nullable

Reply:
Interesting, see https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.Xml/src/System/Xml/Schema/XmlSchema.cs#L69. The return value can only be null if the second parameter is not null and a valid `ValidationEventHandler`.

Reply:
I guess this is resoved then?